packages/vite-extensions/vite-website-cache-paths.ts

```diff
@@ -0,0 +1,93 @@
+import { readdirSync, statSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+/**
+ * We want to cache all files from the website root directory, and the assets directory, except for the files listed below.
+ */
+const patternsToNotCache = [
+	/**
+	 * Static files that are not needed for the website to function offline.
+	 * TODO: Check if these are needed in the production build.
+	 */
+	'/package.json',
+	'/README.md',
+	'/.DS_Store',
+	'/index.cjs',
+	'/index.d.ts',
+	/\/lib\/.*/, // Remote lib files
+	/\/test-fixtures\/.*/, // Test fixtures
+	/**
+	 * WordPress remote assets that are fetched on demand by Playground.
```

Comment:
```suggestion
	 * WordPress assets removed from the minified builds, for example:
	 *
	 *      /wp-6.2/wp-content/themes/twentytwentyone/style.css
	 *
	 * We don't need to cache them, because they're only used in a short time
	 * window after the Playground is initially loaded and before they're all backfilled
	 * from a dedicated ZIP file shipping all the static assets for a given minified 
	 * build.
	 *
	 * See https://github.com/WordPress/wordpress-playground/pull/1531
```